### PR TITLE
Create MasterReach1.json

### PR DESCRIPTION
**Pull request by MasterReach1**
## Changes proposed

Create my LinkFree JSON file.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [n/a] My change requires changes to the documentation.
- [n/a] I have updated the documentation accordingly.
- [n/a] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1788"></a>

### CODE REVIEW

File: public/data/MasterReach1.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "icon": "twitter"
+    }
+   ],
+  "milestones": [
+    {
+      "title": "Started developing games with Python & Blender",
+      "date": "October 2013",
+      "icon": "book",
+      "color": "orange",
+      "description": "Started with Python",
```
Comment (kumarsonsoff3) [inline public/data/MasterReach1.json:24]: Please remove this trailing comma from line no. `24`. 
```suggestion
      "description": "Started with Python"
```

File: public/data/MasterReach1.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "MasterReach1",
+  "type": "personal",
+  "bio": "Web Developer | Ethical Hacker | Open Source Enthusiast",
+  "avatar": "https://github.com/MasterReach1.png",
+  "links": [
+    {
+      "name": "Follow me here on Github!",
```
Comment (kumarsonsoff3) [inline public/data/MasterReach1.json:8]: I recommend capitalizing this `H` in Github, What do you think ⁉

```suggestion
      "name": "Follow me here on GitHub!",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/MasterReach1.json (hunk 1 above)
Comment (MasterReach1) [reply public/data/MasterReach1.json]: I have made this change. Do I need to make another PR or can the changes be seen on this one?

I'm super new to this, so not 100% on how it works.

File: public/data/MasterReach1.json (hunk 2 above)
Comment (MasterReach1) [reply public/data/MasterReach1.json]: I think that's a great idea. I have capitalized the H and it looks way better.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

File: public/data/MasterReach1.json (hunk 1 above)
Comment (Panquesito7) [reply public/data/MasterReach1.json]: This current PR is fine. You don't need to make another PR to make changes in this PR. 😃

**Review by Panquesito7 — APPROVED** [review]
Looks great. Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Awesome, Thanks! 🎉

**Pull request merged**